README.md

```diff
@@ -1,6 +1,142 @@
+
+The installer provides these OS options:
+
+**Asahi Linux Desktop**
+
+A customized remix of Arch Linux ARM that comes with a full Plasma desktop and all the basic packages to get you started with a desktop environment. It includes a graphical first-boot set-up wizard, so you won’t have to dig around to change your settings or create your first user. No root password by default; use sudo to become root.
+
+**Asahi Linux Minimal (Arch Linux ARM)**
+
+A vanilla Arch Linux ARM environment, with only the minimal support packages to integrate with the boot process and hardware on Apple Silicon machines. Arch users will feel right at home!
+
+Log in as root/root or alarm/alarm. Don’t forget to change both passwords! SSH is disabled by default for security reasons, so you’ll have to enable it manually.
+
+**UEFI environment only (m1n1 + U-Boot + ESP)**
+
+No distribution, just a minimal UEFI boot environment. With this, you can boot an OS installer from a USB drive and install whatever you want (as long as it supports these machines, of course)!
+
+
+## What works
+
+All M1, M1 Pro, and M1 Max devices are supported except for the Mac Studio.
+
+- Wi-Fi
+- USB2 (Thunderbolt ports)
+- USB3 (Mac Mini Type A ports)
+- Screen (no GPU)
+- NVMe
+- Lid switch
+- Power button
+- Built-in display (framebuffer only)
+- Built-in keyboard/touchpad
+- Display backlight on/off
+- Battery information / charge control
+- RTC
+- Ethernet (desktops)
+- SD card reader (M1 Pro/Max)
+- CPU frequency switching
+
+
+M1 machines only (no Pro/Max):
+
+- Headphones jack (might be flaky)
+
+Mac Mini only:
+
+- HDMI output
+
+Not yet, but coming soon:
+
+- USB3
+- Speakers
+- Display controller (backlight brightness control, V-Sync, proper DPMS)
+
+## What doesn’t work:
+
+Everything else, but notably:
+
+- DisplayPort
+- Thunderbolt
+- HDMI on the MacBooks
+- Bluetooth
+- GPU acceleration
+- Video codec acceleration
+- Neural Engine
+- CPU deep idle
+- Sleep mode
+- Camera
+- Touch Bar
+  - Note: on the 13" MacBook Pro, you can use Fn + the number row keys (1-9, 0, and the next two) as F1..F12 in lieu of the Touch Bar.
+
+## Known bugs
+
+- If Wi-Fi doesn’t work, try toggling it off and on in the network management menu
+- If the headphones jack doesn’t work or only one channel works, try rebooting. There’s a flakiness issue.
+
+## Known broken applications
+
+The Asahi Linux kernel is compiled to use 16K pages. This both performs better, and is required with our kernel branch right now in order to work properly with the M1’s IOMMUs. Unfortunately, some Linux software has problems running with 16K pages. Most notably:
+
+- Chromium (needs volunteer to fix)
```

Comment:
Chromium is fixed already, see https://github.com/AsahiLinux/docs/wiki/Broken-Software

Reply:
Ok I will add that, make new PR. 

Also, I have a start on the GUI, if you guys could take a look to see if you approve before I work more on it:
<img width="826" alt="Screen Shot 2022-08-04 at 19 30 46" src="https://user-images.githubusercontent.com/71262643/182972203-89c78fa3-6e00-4e32-a91d-05b25dc71131.png">
feel free to make any comments, don't worry about me, I'm already very impressed with the whole project and am honored to be able to contribute!